index.html

```diff
@@ -3,37 +3,378 @@
+                </li>
+                <li class="portfolio__item" onmouseover="showPortfolioHover(this)" onmouseout="hidePortfolioHover(this)">
+                    <div class="portfolio__hover">
+                        <div class="fs1 services__icon portfolio__hover-icon" aria-hidden="true" data-icon="&#xe02e;"></div>
+                        <h1>Illustration</h1>
+                        <h2>Graphic / Illustration / Poster</h2>
+                    </div>
+                    <img src="images/portfolio-img4.jpg" />
+                </li>
+                <li class="portfolio__item" onmouseover="showPortfolioHover(this)" onmouseout="hidePortfolioHover(this)">
+                    <div class="portfolio__hover">
+                        <div class="fs1 services__icon portfolio__hover-icon" aria-hidden="true" data-icon="&#xe02e;"></div>
+                        <h1>Illustration</h1>
+                        <h2>Graphic / Illustration / Poster</h2>
+                    </div>
+                    <img src="images/portfolio-img5.jpg" />
+                </li>
+                <li class="portfolio__item" onmouseover="showPortfolioHover(this)" onmouseout="hidePortfolioHover(this)">
+                    <div class="portfolio__hover">
+                        <div class="fs1 services__icon portfolio__hover-icon" aria-hidden="true" data-icon="&#xe02e;"></div>
+                        <h1>Illustration</h1>
+                        <h2>Graphic / Illustration / Poster</h2>
+                    </div>
+                    <img src="images/portfolio-img6.jpg" />
+                </li>
+            </ul>
+        </section>
+
+        <section id="blog" class="blog" onmouseover="highlightItem(5)" onmouseout="removeHighlightItem(5)">
+            <h1 class="section-title">Our Blog</h1>
+            <p class="section-description">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+            </p>
 
+            <ul class="blog__list">
+                <li class="blog__item expanded">
+                    <h3>Online Business / Dec 9, 2016</h3>
+                    <h1>The Best Way to make Online Business</h1>
+                    <h2>By Johnathan Doe</h2>
+                </li>
+                <li class="blog__item expanded">
+                    <h3>Online Business / Dec 9, 2016</h3>
+                    <h1>The Best Way to make Online Business</h1>
+                    <h2>By Johnathan Doe</h2>
+                </li>
+                <li class="blog__item">
+                    <h3>Online Business / Dec 9, 2016</h3>
+                    <h1>The Best Way to make Online Business</h1>
+                    <h2>By Johnathan Doe</h2>
+                </li>
+                <li class="blog__item">
+                    <h3>Online Business / Dec 9, 2016</h3>
+                    <h1>The Best Way to make Online Business</h1>
+                    <h2>By Johnathan Doe</h2>
```

Comment:
headers are used to create a map of the app, they are not meant to be used if they match the styles